### PR TITLE
Generate bindings for each target

### PR DESCRIPTION
**Pull request by Octogonapus**
As far as I understand it, we need to generate bindings per-target because the JLL contains header files per-target and they could differ.
I would rather be safe than sorry.

### CODE REVIEW

**Comment by quinnj** [comment]
Hmmmm, yeah, I agree we should do the platform specific generated code. I wonder if there's a relatively simple way we could only put truly platform-specific code in the individual platform files and share the other 99%, partly to avoid needing to commit 145K lines of code 😬

File: gen/Manifest.toml [hunk 1]
```diff
@@ -209,9 +209,9 @@ version = "1.2.13+1"
 
 [[deps.aws_c_common_jll]]
 deps = ["Artifacts", "JLLWrappers", "Libdl"]
-git-tree-sha1 = "586a1aa2fdda172c45587b52f5329fedde14f44c"
+git-tree-sha1 = "3cbdb09867f78386555469ddecaf276ff327936a"
 uuid = "73048d1d-b8c4-5092-a58d-866c5e8d1e50"
-version = "0.9.14+0"
+version = "0.9.3+1"
```
Comment (quinnj) [inline gen/Manifest.toml]: Any reason in particular to downgrade here? I'd like to keep things as up-to-date as possible

Comment (Octogonapus) [reply gen/Manifest.toml]: no. oops

**Comment by Octogonapus** [comment]
@quinnj This is RTM for me. I had to ignore some ITT API symbols that aren't useful to us.

File: gen/Manifest.toml (hunk 1 above)
Comment (quinnj) [inline gen/Manifest.toml]: looks like this slipped back again

Comment (quinnj) [reply gen/Manifest.toml]: Ok, I bumped it back up and re-ran the generation, which was good for me to try out locally anyway.

Comment (Octogonapus) [reply gen/Manifest.toml]: Thanks, not sure how that happened

**Pull request merged**